### PR TITLE
add hint that config permission problem is likely due to wrong user

### PR DESCRIPTION
**Pull request by davewhiteland**
see #40 for user running fixmystreet not as 'fms' user for example,
so I'm suggesting rewording error message to make it clear this
mightn't be a problem with the config file itself

### CODE REVIEW

**Comment by dracos** [comment]
Could you have this without the permissions change of the file? And also, I'd reword the first half of the message too, it's rubbish in general!

**Comment by davewhiteland** [comment]
oof permission change was entirely accidental (as you may have guessed), fixed and force-pushed so it's not visible on this branch now.

Message rewritten too, OK now?

**Comment by dracos** [comment]
Looks good :)
